addons/binding/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/handler/KeContactHandler.java

```diff
@@ -549,6 +529,23 @@ protected void onRead(ByteBuffer byteBuffer, DatagramChannel datagramChannel) {
                         Map<String, String> properties = editProperties();
                         properties.put(CHANNEL_FIRMWARE, entry.getValue().getAsString());
                         updateProperties(properties);
+                        if (entry.getValue().getAsString().contains("2.01m21")) {
```

Comment:
An enum is suitable to store the combination. So you can store the substring in the enum an provide a static lookup method in the enum, it keeps the code way more localized. See for example https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/RFXComValueSelector.java